deps/rabbit/src/rabbit_db_exchange.erl

```diff
@@ -711,13 +710,20 @@ delete_all_in_khepri_tx(VHostName) ->
     {ok, NodeProps} = khepri_tx_adv:delete_many(Pattern),
     Deletions =
     maps:fold(
-      fun(_Path, #{data := X}, Deletions) ->
-              {deleted, #exchange{name = XName}, Bindings, XDeletions} =
-                rabbit_db_binding:delete_all_for_exchange_in_khepri(
-                  X, false, true),
-              Deletions1 = rabbit_binding:add_deletion(
-                             XName, X, deleted, Bindings, XDeletions),
-              rabbit_binding:combine_deletions(Deletions, Deletions1)
+      fun(Path, Props, Deletions) ->
+              case {Path, Props} of
+                  {?RABBITMQ_KHEPRI_EXCHANGE_PATH(VHostName, _),
+                   #{data := X}} ->
```

Comment:
Mostly a note to myself: I wonder if `?RABBITMQ_KHEPRI_EXCHANGE_PATH()` can be use in the function clause directly instead of having a `case`.

Reply:
I was hopeful we could do that but we would need https://github.com/erlang/eep/blob/master/eeps/eep-0055.md to be implemented - we would want to pin bindings like `^VHostName`. Without EEP55 I believe we can't do that, it would be a compile error to re-use the binding if I remember correctly